lib/application/presentation/login_screen.dart

```diff
@@ -0,0 +1,257 @@
+
+  @override
+  void dispose() {
+    emailController.dispose();
+    passwordController.dispose();
+
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        body: Container(
+          decoration: const BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage(MyIcons.authBackground),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Padding(
+            padding: const EdgeInsets.only(left: 40, right: 40, bottom: 40),
+            child: ListView(
+              children: [
+                const SizedBox(
+                  height: 50,
+                ),
+                Center(
+                  child: Column(
+                    children: [
+                      Image.asset(MyIcons.categoryIcons['all']!),
+                      Text(
+                        'Sign In',
+                        style: Theme.of(context).textTheme.bodyLarge!.copyWith(fontSize: 30),
+                      ),
+                      const SizedBox(height: 8),
+                      Text(
+                        'Welcome!',
+                        style: Theme.of(context).textTheme.bodyLarge!.copyWith(fontWeight: FontWeight.w400),
+                      ),
+                    ],
+                  ),
+                ),
+                const SizedBox(height: 40),
+                TextField(
+                  controller: emailController,
+                  cursorHeight: 24,
+                  cursorColor: MyColors.colorFFFFFF,
+                  textInputAction: TextInputAction.next,
+                  decoration: InputDecoration(
+                    labelText: 'Email',
+                    labelStyle: Theme.of(context).textTheme.bodyLarge!.copyWith(color: MyColors.colorFB65BA),
+                    enabledBorder: const UnderlineInputBorder(
+                      borderSide: BorderSide(width: 1.5, color: MyColors.color9B9A9B),
+                    ),
+                  ),
+                ),
+                const SizedBox(
+                  height: 20,
+                ),
+                TextField(
+                  controller: passwordController,
+                  cursorHeight: 24,
+                  cursorColor: MyColors.colorFFFFFF,
+                  textInputAction: TextInputAction.done,
+                  decoration: InputDecoration(
+                    labelText: 'Password',
+                    labelStyle: Theme.of(context).textTheme.bodyLarge!.copyWith(color: MyColors.colorFB65BA),
+                    enabledBorder: const UnderlineInputBorder(
+                      borderSide: BorderSide(width: 1.5, color: MyColors.color9B9A9B),
+                    ),
+                  ),
+                  obscureText: true,
+                ),
+                const SizedBox(height: 40),
+                Center(
+                  child: Column(
+                    children: [
+                      ElevatedButton(
```

Comment:
Ovakav stil buttona na nekoliko mjesta u aplikaciji koristite, napravite onda svoj Widget pa mu kao parametere možete napraviti: label, padding, onPressed itd. što je već drugačije